src/style.css

```diff
@@ -4,49 +4,63 @@
   box-sizing: border-box;
 }
 
+:root {
+  --text: #0b140e;
+  --background: #deede4;
+  --primary: #2a4c48;
+  --secondary: #c0d2dd;
+  --accent: #4a7087;
+}
+
 html {
   scroll-behavior: smooth;
 }
 
 body {
   font-family: 'Poppins', 'Times New Roman', Times, serif;
+  color: var(--text);
+  background-color: var(--background);
 }
 
 .container {
   width: 90%;
   display: flex;
-  flex-direction: row-reverse;
+  flex-direction: column;
   justify-content: center;
-
-  /* align-items: center; */
   padding: 1rem;
   margin: 3rem auto;
   gap: 1.25rem;
 }
 
 #form-box {
   margin-top: 2rem;
-  width: 40%;
+  width: 100%;
   display: flex;
   flex-direction: column;
-  align-items: flex-end;
+  align-items: center;
+  justify-content: center;
   gap: 1rem;
 }
 
 .form-side {
```

Comment:
- [ ] As required for the project your application to be properly styled and its layout looks like the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), I think it would be better if better if you update the header styling.
### Wireframe Styling
![Screenshot from 2023-09-01 03-39-21](https://github.com/Chimoneg27/Leaderboard-With-API/assets/83042742/44106937-db15-482d-98b0-322f3a68551e)

### Your Styling
![Screenshot from 2023-09-01 03-46-55](https://github.com/Chimoneg27/Leaderboard-With-API/assets/83042742/4e9f3d37-1c5e-4fb9-a59e-5d4c8d40e2df)